jwst/datamodels/schemas/lev3_prod.schema.yaml

```diff
@@ -4,18 +4,15 @@ properties:
     type: object
     properties:
       resample:
-        # 2019-06-12: Title has changed to "Resampling parameter information" in keyword dictionary.
-        title: Metadata describing resampling done using this data
+        title: Resampling parameter information
```

Comment:
All of the updates to this schema look fine, but I'm wondering if it's finally time to do something with reworking this whole schema. First, the name is not appropriate, because it isn't confined to use with level-3 products, but is more generally applicable to any drizzled product. So at the very least we should rename the schema file to something more generic, like "driz_prod.schema.yaml". Unfortunately there's already a slightly higher-level schema called "drizproduct.schema.yaml", which inherits this schema. I wonder if the contents of "lev3_prod.schema" can just be merged into "drizproduct.schema"?

@jdavies-st do you have any thoughts/suggestions about this?

Reply:
Let's just keep this as-is for now and can always do refactoring later.